nuxt-frontend/components/frontend-development/DevelopmentSection.vue

```diff
@@ -0,0 +1,194 @@
+<template>
+  <section
+    class="my-16 lg:mt-60 lg:mb-0 relative 3xl:h-[280vh] 3xl:outer-container"
+  >
+    <div
+      class="flex flex-col gap-4 lg:gap-6 container text-center lg:sticky top-28 3xl:!pb-0"
+      id="parentContainer"
+    >
+      <h2 class="text-black-87 mobile-header-2 lg:desk-header-2">
+        Frontend <br class="md:hidden" />development services
+      </h2>
+      <p
+        class="sub-h1-regular lg:mobile-header-2-regular text-black-60 xs:px-2 2xl:!px-32"
+      >
+        At Canopas, we understand the power of a stellar frontend interface. We
+        believe that a potent blend of aesthetics, responsiveness, and intuitive
+        design is paramount to captivate your audience and drive engagement. We
+        aim to create digital platforms that don't just tell your brand's story
+        but also offer an unparalleled user experience.
+      </p>
+    </div>
+    <!-- Mobile Swiper UI starts-->
+    <div class="swiper-content mt-6 lg:hidden">
+      <swiper
+        :slidesPerView="1"
+        :centeredSlides="true"
+        :spaceBetween="10"
+        :modules="modules"
+        :autoplay="{
+          delay: 3000,
+          disableOnInteraction: false,
+        }"
+        :speed="1500"
+        :pagination="pagination"
+        :breakpoints="{
+          '540': {
+            slidesPerView: 1.2,
+            spaceBetween: 0,
+          },
+          '620': {
+            slidesPerView: 1.4,
+            spaceBetween: 0,
+          },
+          '700': {
+            slidesPerView: 1.5,
+          },
+        }"
+        class="swiper-container"
+      >
+        <swiper-slide
+          v-for="(item, index) in items"
+          :key="index"
+          class="cursor-pointer px-4"
+        >
+          <div
+            :class="{
+              'bg-pink-40': index % 2 === 0,
+              'bg-white-smoke': index % 2 !== 0,
+            }"
+            class="container py-6 rounded-xl"
+          >
+            <h3 class="mobile-header-3-semibold text-black-87">
+              {{ item.title }}
+            </h3>
+            <div class="sub-h3-regular text-black-60 mt-4">
+              <span
+                v-html="item.description"
```

Comment:
`<span>` within `<span>` is not a good structure.
You can directly set the description into div, no need to use span for it.